src/python/grpcio_tests/tests/channelz/_channelz_page_test.py

```diff
@@ -0,0 +1,212 @@
+# Copyright 2019 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests of Serving HTTPS Channelz Page."""
+
+import unittest
+import time
+import logging
+import re
+
+import requests
+
+import grpc
+from grpc_channelz.v1 import channelz
+
+from tests.unit import test_common
+from tests.unit.framework.common import test_constants
+
+# Matches first link inside of a table
+_table_a_href_re = re.compile(r'<table.+?<a href=[\'"](.+?)[\'"]', re.DOTALL)
```

Comment:
HTML parser doesn't require `lxml`, so it can be use in our environment.
Sadly, its semantics is too simple to fulfill our need. We have to add a state machine with HTMLParser to locate the information we want. What do you think...?